source/opt/optimizer.cpp

```diff
@@ -77,7 +77,8 @@ Optimizer& Optimizer::RegisterPerformancePasses() {
       .RegisterPass(CreateBlockMergePass())
       .RegisterPass(CreateLocalMultiStoreElimPass())
       .RegisterPass(CreateInsertExtractElimPass())
-      .RegisterPass(CreateCommonUniformElimPass())
+      // Currently causing driver crashes
```

Comment:
Could you reference the issue? It's #946, IIRC.

Reply:
Done.